lib/posix/options/stropts.c

```diff
@@ -18,3 +18,16 @@ int putmsg(int fildes, const struct strbuf *ctlptr, const struct strbuf *dataptr
 	errno = ENOSYS;
 	return -1;
 }
+
+int putpmsg(int fildes, const struct strbuf *ctlptr,
+const struct strbuf *dataptr, int band, int flags)
```

Comment:
format is off

Reply:
Do I need to break a long line? I did the warning from my IDE that this line is too long.